rsr_convert_scripts/abi_rsr.py

```diff
@@ -69,27 +67,21 @@ def __init__(self, bandname, platform_name):
         self.filename = self.requested_band_filename
 
     def _load(self, scale=1.0):
-        """Load the ABI relative spectral responses
-        """
-
+        """Load the ABI relative spectral responses."""
         LOG.debug("File: %s", str(self.requested_band_filename))
 
         data = np.genfromtxt(self.requested_band_filename,
                              unpack=True,
-                             names=['wavelength',
-                                    'wavenumber',
-                                    'response'],
                              skip_header=2)
-
-        wvl = data['wavelength'] * scale
-        resp = data['response']
+        wvl = data[0] * scale
+        resp = data[2]
 
         self.rsr = {'wavelength': wvl, 'response': resp}
 
 
 def main():
-    """Main"""
-    for platform_name in ['GOES-16', 'GOES-17', ]:
+    """Create pyspectral hdf5 RSR files for all GOES sats."""
+    for platform_name in ['GOES-18', 'GOES-19', ]:
```

Comment:
Why not all of GOES-16, -17, -18, -19 (also for historical reference)?

Reply:
Good point, done! Thx